README.md

```diff
@@ -119,14 +121,16 @@ tkn pipeline start operator-hosted-pipeline \
   --param env=prod \
   --param preflight_min_version=0.0.0 \
   --param ci_min_version=0.0.0 \
+  --param hydra_secret_name=hydra-credentials \
+  --param hydra_secret_username_key=username \
+  --param hydra_secret_password_key=password \
```

Comment:
In the interest of keeping these instructions clean and easier to get started with, I suggest we don't specify params for which the default value follows the setup instructions. Now that we have the ability to deploy our own custom envs, only a power user would need to mess with these. Wdyt?

Reply:
Sounds good. They're not really needed and does clutter up the command quite a bit. Updated.